src/pybind/mgr/dashboard/frontend/src/app/shared/components/usage-bar/usage-bar.component.ts

```diff
@@ -1,53 +1,115 @@
-import { Component, Input, OnChanges } from '@angular/core';
-
-import _ from 'lodash';
+import { Component, Input, ElementRef, OnInit } from '@angular/core';
+import { ChartTabularData, MeterChartOptions, Statuses } from '@carbon/charts-angular';
+import { MeterChartColor } from '../../enum/css-style-variable.enum';
+import { CssHelper } from '../../classes/css-helper';
+import { DimlessBinaryPipe } from '../../pipes/dimless-binary.pipe';
+import { DimlessPipe } from '../../pipes/dimless.pipe';
 
 @Component({
   selector: 'cd-usage-bar',
   templateUrl: './usage-bar.component.html',
   styleUrls: ['./usage-bar.component.scss']
 })
-export class UsageBarComponent implements OnChanges {
-  @Input()
-  total: number;
-  @Input()
-  used: any;
-  @Input()
-  warningThreshold?: number;
-  @Input()
-  errorThreshold?: number;
-  @Input()
-  isBinary = true;
-  @Input()
-  decimals = 0;
-  @Input()
-  calculatePerc = true;
-  @Input()
-  title = $localize`usage`;
-  @Input()
-  customLegend?: string;
-  @Input()
-  customLegendValue?: string;
-  @Input()
-  showFreeToolTip = true;
-  @Input()
-  showMultisiteTooltip = false;
-
-  usedPercentage: number;
-  freePercentage: number;
-
-  ngOnChanges() {
-    if (this.calculatePerc) {
-      this.usedPercentage = this.total > 0 ? (this.used / this.total) * 100 : 0;
-      this.freePercentage = 100 - this.usedPercentage;
-    } else {
-      if (this.used) {
-        this.used = this.used.slice(0, -1);
-        this.usedPercentage = Number(this.used);
-        this.freePercentage = 100 - this.usedPercentage;
-      } else {
-        this.usedPercentage = 0;
+export class UsageBarComponent implements OnInit {
+  @Input() total: number = 0;
+  @Input() used: number = 0;
+  @Input() title: string = $localize`Usage`;
+  @Input() warningThreshold: number = 0.9;
+  @Input() errorThreshold: number = 0.95;
```

Comment:
```suggestion
  @Input() warningThreshold: number = 90; // percentage
  @Input() errorThreshold: number = 95; // percentage
```
Now that we have moved to meter chart we can directly used the percentages, and pass that.
No need to convert later